test/helpers/kubectl.go

```diff
@@ -136,6 +139,16 @@ var (
 		"nodeinit.enabled":       "true",
 	}
 
+	eksENIHelmOverrides = map[string]string{
+		"egressMasqueradeInterfaces": "eth0",
+		"eni":                        "true",
+		"ipam.mode":                  "eni",
+		"ipv6.enabled":               "false",
+		"k8s.requireIPv4PodCIDR":     "false",
```

Comment:
Is this necessary if we also set `ipam.mode=eni`?

Reply:
@joestringer -- yes we need to set that to false. It gets set as `true` via https://github.com/cilium/cilium/blob/master/test/helpers/kubectl.go#L101. In the daemon config it defaults to false so that is why in the ENI install instructions there is no mention of it.